projects/packages/publicize/src/class-publicize.php

```diff
@@ -442,49 +505,20 @@ public function unglobalize_connection( $connection_id ) {
 		}
 	}
 
-	/**
-	 * As Jetpack updates set the refresh transient to a random amount
-	 * in order to spread out updates to the connection data.
-	 *
-	 * @param string $version The Jetpack version being updated to.
-	 */
-	public function init_refresh_transient( $version ) {
-		if ( version_compare( $version, '10.2.1', '>=' ) && ! get_transient( self::CONNECTION_REFRESH_WAIT_TRANSIENT ) ) {
-			$this->set_refresh_wait_transient( wp_rand( 10, HOUR_IN_SECONDS * 24 ) );
-		}
-	}
-
 	/**
 	 * Grabs a fresh copy of the publicize connections data.
-	 * Only refreshes once every 12 hours or retries after an hour with an error.
+	 * Only refreshes once every 10 hours or retries after 10 mins in case an error.
```

Comment:
I think this will retry immediately if there's an error, we're not wating 10 minutes.

Reply:
Updated the comment